api/v1alpha1/openapi.yaml

```diff
@@ -3088,14 +3088,14 @@ components:
     DeviceDecommission:
       type: object
       properties:
-        decommissionTarget:
+        target:
```

Comment:
That'd also remove the double "decommission" alright, but what meant wasn't changing the property name.
If you were to define a separate type like you did [here](https://github.com/flightctl/flightctl/blob/main/api/v1alpha1/openapi.yaml#L3937-L3948) and referenced that here, you'd be able to get clean variable and type names.

Reply:
We could probably revert to `decommissionTarget` now, though I don't have a strong preference. The name should just be intuitive.

Reply:
It looks like this when generated:
```
const (
	DeviceDecommissionTargetTypeFactoryReset DeviceDecommissionTargetType = "FactoryReset"
	DeviceDecommissionTargetTypeUnenroll     DeviceDecommissionTargetType = "Unenroll"
)
```
which I actually think is a bit nicer than the original one.

Reply:
Right. But that's for developers and to avoid naming conflicts later.
Through the change you've made, both `target` and `decommissionTarget` would now produce the above code, so now we can choose what's the most intuitive.